### PR TITLE
zpoline integration for an option to hijack library

### PR DESCRIPTION
**Pull request by thehajime**
this is still under progress but we've been working on another implementation of hijack library.  More to come soon and will publish as (non-draft) a pull request here.

https://github.com/yasukata/zpoline

### CODE REVIEW

**Comment by thehajime** [comment]
sorry to be late.  the patchset is ready for review and please take a look at it.  thanks !

File: Documentation/lkl.txt [hunk 1]
```diff
@@ -460,6 +460,71 @@ The following are the list of keys to describe a JSON file.
...
+The LKL hijack library works together with zpoline by loading LKL.
+
+zpoline currently only works on x86_64 machines.
+
+To use the zpoline-enabled hijack library, please follow the
+instruction below.
+
+- Build
+```
+make -C tools/lkl -j8 zpoine=../zpoline
```
Comment (AkihiroSuda) [inline Documentation/lkl.txt:479]: typo: zpoine

Comment (thehajime) [reply Documentation/lkl.txt]: thanks, fixed.

**Review by tavip — APPROVED** [review]
LGTM, nice work!

**Pull request merged**